docs/source/gssapi.rst

```diff
@@ -65,6 +65,34 @@ imported in the high-level API :mod:`gssapi` module:
 .. autoclass:: gssapi.RequirementFlag
     :show-inheritance:
 
+The ``ok_as_delegate`` flag corresponds to the C level flag
+``GSS_C_DELEG_POLICY_FLAG``. This flag is similar to ``delegate_to_peer``
+except it only delegates if the KDC delegation policies for the service
+principal allow it to use delegation. This is typically used on Microsoft
+domain environments to control whether constrained or unconstrained delegation
+is allowed for a service principal. By setting this flag, the delegation
+process follows the same behaviour as delegation on SSPI/Windows.
+
+Here are the four cases when either of these flags are set or not.
+
+Neither flag set
+   No delegation occurs.
+
+delegate_to_peer
+   Always try to delegate regardless of the KDC delegation policies.
+   ``delegate_to_peer`` is set in the return flags if successful.
+
+ok_as_delegate
```

Comment:
This scenario does not work as documented on MIT. Trying to get some clarification from the krb5 devs to see what they say before updating the docs.

Reply:
Turns out I was doing some faulty testing, after trying again I found that MIT was acting just like Heimdal and what was in the documents. No idea what I did wrong the first time but it's good they act the same :)